static/sass/_snapcraft_release.scss

```diff
@@ -59,4 +67,24 @@
     right: 0;
     top: 10px;
   }
+
+  .p-channel-buttons {
+    display: inline-block;
+    margin-right: .5rem;
+    width: 2rem;
+  }
+
+
```

Comment:
empty line

Reply:
If only linter would catch things like that. I'll fix that with next release UI updates.